lib/src/asn1/liblte_mme.cc

```diff
@@ -3019,6 +3020,10 @@ LIBLTE_ERROR_ENUM liblte_mme_unpack_emergency_number_list_ie(uint8**
     emerg_num_list->N_emerg_nums = 0;
     while (length < sent_length) {
       idx                                               = emerg_num_list->N_emerg_nums;
+      if(idx >= LIBLTE_MME_EMERGENCY_NUMBER_LIST_MAX_SIZE) {
```

Comment:
please check the format, a space is missing there, it should be
`if (idx >= LIBLTE_MME_EMERGENCY_NUMBER_LIST_MAX_SIZE) {`

Reply:
OK, I'll do like what you say.